contracts/transfer/circuits/src/execute/output.rs

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_plonk::jubjub::JubJubExtended;
+use phoenix_core::Note;
+
+use dusk_plonk::prelude::*;
+
+#[derive(Debug, Clone)]
+pub struct CircuitOutput {
```

Comment:
The name might be confusing since this does not impl `Circuitbuilder`

Reply:
This is an output note that can be used by the gadget of the circuit. This is not a circuit by itself, but the output representation for the `ExecuteCircuit`